SoftLayer/CLI/autoscale/delete.py

```diff
@@ -0,0 +1,28 @@
+"""Delete autoscale."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.managers.autoscale import AutoScaleManager
+
+
+@click.command()
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Sets a user's status to CANCEL_PENDING, which will immediately disable the account,
+
+    and will eventually be fully removed from the account by an automated internal process.
+
+    Example: slcli user delete userId
```

Comment:
Update the command description and change example by `slcli autoscale delete autoscaleId`